docs/source/cli_help/config.rst

```diff
@@ -71,15 +54,12 @@ Note: ``config`` is not a command as such, just a help topic (there is no ``git
 
 **Environment variables:**
 
-    * ``GIT_MACHETE_EDITOR``
-
-        Name of the editor used by ``git machete e[dit]``, example: ``vim`` or ``nano``.
-
-    * ``GIT_MACHETE_REBASE_OPTS``
-
-        Used to pass extra options to the underlying ``git rebase`` invocation (called by the executed command, such as: ``reapply``, ``slide-out``, ``traverse``, ``update``)
-        Example: ``GIT_MACHETE_REBASE_OPTS="--keep-empty --rebase-merges" git machete update``.
+``GIT_MACHETE_EDITOR``
```

Comment:
Is the formatting consistent with other .rst files where env vars are documented?

Reply:
Yes, now it is